app/views/layouts/_navbar.html.haml

```diff
@@ -0,0 +1,52 @@
+.container
+.navbar-header
```

Comment:
@PenneyGadget So the .navbar-header should be in one level inside of the .container div. The way it currently is setup the container div is just <div class="container"></div>